posts/ja/2023-02-13-22.0.0.13.adoc

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: "Open Liberty 22.0.0.13における最大FFDC期間の設定とCVEやその他主なバグの修正"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/mbroz2
+author_github: https://github.com/mbroz2
+seo-title: Open Liberty 22.0.0.13では、CVEやその他の重要なバグ修正とともに、FFDCの最大期間を設定可能 - OpenLiberty.io
+seo-description: Open Liberty 22.0.0.13 では、FFDC ログファイルが設定された期間に達すると、自動的に消去される機能が提供されます。このリリースでは、gRPCとgRPCクライアントのCVEに対処した2つのバグを含む、多くの重要なバグ修正も提供されます。
+blog_description: Open Liberty 22.0.0.13 では、FFDC ログファイルが設定された期間に達した後、自動的に消去する機能が提供されます。このリリースでは、gRPCとgRPCクライアントのCVEに対処した2つのバグを含む、多くの重要なバグ修正も提供されます。
+blog-available-in-languages:
+- lang: en
+  path: /blog/2022/12/20/22.0.0.13.html
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= Open Liberty 22.0.0.13では、CVEやその他の重要なバグ修正とともに、FFDCの最大使用期間を設定することが可能です。
+Michal Broz <https://github.com/mbroz2>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+オリジナル・ブログ: link:{url-prefix}/blog/2022/12/20/22.0.0.13.html[Configurable maximum FFDC age along with CVE and other notable bug fixes in Open Liberty 22.0.0.13]
+
+翻訳: link:{url-prefix}https://github.com/kaori-asa[浅田 かおり] 
+
+Open Liberty 22.0.0.13 では、FFDC ログファイルが設定された期間に達した後、自動的に消去する機能が提供されます。このリリースでは、gRPC と gRPC クライアントの CVE に対処した 2 つのバグを含む、多くの重要なバグ修正も提供されます。 gRPC と WebSocket のトピックをカバーする 2 つの新しいガイドも、私たちの広範なガイドのリストに加わっています。
+Jakarta EE 10、MicroProfile 6、InstantOn など、私たちが現在進行中の機能や特徴に興味がある場合は、最近の link:https://openliberty.io/blog/?search=beta&key=tag[ベータ版ブログ記事] をご覧ください。
+
+
+In link:{url-about}[Open Liberty] 22.0.0.13:
+
+* <<maxFfdcAge, 設定可能な最大 FFDC 期間>> 
+* <<CVEs, セキュリティ脆弱性(CVE)の修正>>
+* <<bugs, 重要なバグ修正>>
+
+Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
```

Comment:
Not sure if this sentence is the same as the translated one below and needs to be deleted (did a quick translate and they seem to convey the same thing) or a sentence that needs to be translated.

Reply:
HI, @steven1046  @natalie-bernhard @kinueng  

Sorry I did not notice this sooner!!! 

Thanks again for catching.  Both are translated in Japanese.  Will you remove the following lines please?  
>> Along with the new features and functions added to the runtime, we’ve also made <<guides, updates to our guides>>.
>> The issue has been resolved by declaring `OidcClientImpl` dependent on `SecurityService` and the NPE no longer occurs.

I was contacted by the marketing and techsales team that they have Webinar tomorrow night,  showcasing Liberty's upcoming features (JakartaEE10, MP6, InstantOn, etc)  They are planning to highlight the Japanese blog.  They would like this article published by then.